stubs/defaultConfig.stub.js

```diff
@@ -172,6 +172,7 @@ module.exports = {
       none: 'none',
     },
     caretColor: (theme) => theme('colors'),
+    accentColor: (theme) => theme('colors'),
```

Comment:
Should probably include `auto` as a value here in addition to all of the colors:

https://developer.mozilla.org/en-US/docs/Web/CSS/accent-color#values

Reply:
Added